app/models/application_record.rb

```diff
@@ -1,26 +1,16 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
 
-  def formatted_price
-    price = unit_price/100.00
-    formatted_amount = sprintf("$%.2f", price)
-    if formatted_amount.length > 7
-      formatted_amount = formatted_amount.gsub!(/(\d)(?=(\d{3})+(?!\d))/, "\\1,")
-    end
-    formatted_amount
-  end
-
-  def formatted_revenue
-    revenue = total_revenue/100.00
-    formatted_amount = sprintf("$%.2f", revenue)
+  def format_money(value)
```

Comment:
I'm going to play with this one in a spike branch to see if we can clean it up a little more